src/services/lansweeper/baseRequest.ts

```diff
@@ -27,6 +27,8 @@ const baseRequest: Request = async (client, {
     headers: {
       "Authorization": `Bearer ${isAdmin ? tokens.access_token : placeholders.ACCESS_TOKEN}`,
       "Accept": "application/json",
+      "x-ls-integration-id": INTEGRATION_ID,
+      "x-ls-integration-version": "1.0",
```

Comment:
This should be the version from the `package.json` so that they can more accurately check who is doing what.

Reply:
done